spec/helpers/medication_refill_helper_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe MedicationRefillHelper, type: :helper do
+  include CalendarHelper
+  include Devise::Test::ControllerHelpers
+
+  describe '#save_refill_to_google_calendar' do
+    let(:user) { FactoryBot.create(:user1) }
+    let(:medication) { FactoryBot.create(:medication, user_id: user.id) }
+    let(:exception_text) { 'RESCUE INVOKED' }
+
+    before do
+      allow_any_instance_of(helper.class).to receive(:return_to_sign_in).and_return(exception_text)
+      sign_in user
+    end
+
+    context 'when current_user.google_oauth2_enabled? && new_cal_refill_reminder_needed?(medication)' do
```

Comment:
I would write this out as a sentence instead of referring to methods in the context

Reply:
I agree. I'll fix it in a few hours